.circleci/config.yml

```diff
@@ -47,6 +47,7 @@ jobs:
             export AIRTABLE_TABLE_NAMES=${AIRTABLE_TABLE_NAMES_INTEGRATION}
             export ADDRESSES_API_URL=${ADDRESSES_API_URL}
             export ADDRESSES_API_KEY=${ADDRESSES_API_KEY}
+            export HOTJAR_ID=${HOTJAR_ID}
```

Comment:
I'm still getting my head around the split configuration - I see no reason why this shouldn't work, but if it doesn't, that'd be my first suspicion!